build-tools/api-merge/ApiDescription.cs

```diff
@@ -202,9 +202,11 @@ public void Merge (string apiLocation)
 		public void Save (string filename)
 		{
 			FixupOverrides ();
-			if (filename != null)
+			if (filename != null) {
+				filename = filename.Replace ('\\', Path.DirectorySeparatorChar);
```

Comment:
I think this might be done earlier so that we see correct paths in the log output?

Reply:
Good call!